templates/types/streaming/fastapi/pyproject.toml

```diff
@@ -14,7 +14,7 @@ fastapi = "^0.109.1"
 uvicorn = { extras = ["standard"], version = "^0.23.2" }
 python-dotenv = "^1.0.0"
 aiostream = "^0.5.2"
-llama-index = "0.10.58"
+llama-index = "0.11.1"
```

Comment:
any other changes we could do with the new 0.11 release?

Reply:
From the release notes i don't see any other changes needed. So far, we can start adding `is_empty` filter for llama_cloud and using instrumentation instead of callback module from v0.11